src/python/foglamp/core/http_server.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" aiohttp multi app instances wrapper
+
+https://stackoverflow.com/questions/44850701/multiple-aiohttp-applications-running-in-the-same-process
+Added host to constructor
+"""
+
+import asyncio
+from aiohttp import web
+
+
+class AppWrapper:
```

Comment:
I would ideally like to see more documentation in here as this was clearly an issue that caused problems. From a maintenance perspective it is crucial that this is well understood